### PR TITLE
Alphabetize the contributors list

### PR DESCRIPTION
**Pull request by jsumners**
This PR organizes the contributors list into alphabetical order by surname.

@cagataycali does this list yours in the correct position? I must admit, I do not know how to sort the `Ç` with respect to the English alphabet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,25 +31,25 @@
   "author": "Matteo Collina <hello@matteocollina.com>",
   "contributors": [
     {
-      "name": "Tomas Della Vedova",
-      "url": "http://delved.org"
-    },
```
Comment (mcollina) [inline package.json]: Tomas was completely removed.

**Comment by jsumners** [comment]
The list is alphabetized by surname because that's how you alphabetize names in English. See the section on names in https://simple.wikipedia.org/wiki/Alphabetical_order

File: package.json (hunk 1 above)
Comment (jsumners) [reply package.json]: Not so. It is second in the list. Although, I have learned that the name should probably be alphabetized by "Vedova" instead of "Della".

**Comment by lmammino** [comment]
Thanks for the clarification @jsumners. I'll take care of updating the sorting logic in the website once this is done.

**Comment by jsumners** [comment]
BTW, I wish I could have found a better citation for the rule, but evidently I don't know how to search for that information.

**Comment by lmammino** [comment]
Also, should we keep the `package.json` in sync with the actual list of contributor displayed on the website and on the README?

**Comment by jsumners** [comment]
@lmammino yes, but I don't have the information to fill in the gap there.

File: package.json (hunk 1 above)
Comment (delvedor) [reply package.json]: Nope, my surname is "Della Vedova" :P

File: package.json [hunk 2]
```diff
@@ -31,25 +31,29 @@
   "author": "Matteo Collina <hello@matteocollina.com>",
   "contributors": [
     {
+      "name": "Tommaso Allevi",
+      "email": "tomallevi@gmail.com"
+    },
+    {
```
Comment (mcollina) [inline package.json]: I would prefer for Tomas to be the 1st of the contributors, if you are ok with it.

Comment (allevo) [reply package.json]: The wrong Tom :smile:

Comment (jsumners) [reply package.json]: I get it. I wish package.json supported more than a single author. It definitely needs more support for a large project developed by multiple people.

**Comment by delvedor** [comment]
Is there anything else we can do here?

**Comment by jsumners** [comment]
@delvedor other than re-position your entry in `package.json`, not that I'm aware of.

**Comment by jsumners** [comment]
I have moved @delvedor back to the top of the contributors list in `package.json`. I also added a `"author": true` property to his entry to convey why its not in alphabetical order.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Handler Context Tests",
+      "program": "${workspaceFolder}/test/handler-context.test.js"
+    }
+  ]
+}
```
Comment (delvedor) [inline .vscode/launch.json]: Is this file needed?

Comment (jsumners) [reply .vscode/launch.json]: How did that get in there?

Comment (jsumners) [reply .vscode/launch.json]: Fixed.

**Comment by cagataycali** [comment]
@jsumners, you right, ‘Ç’ isn’t English. I’ll replace to ‘C’

**Comment by delvedor** [comment]
Can we merge this?

**Comment by jsumners** [comment]
I think so. But I was waiting on a LGTM or two.
